api/src/database/migrations/20231009A-update-csv-fields-to-text.ts

```diff
@@ -0,0 +1,47 @@
+import type { Knex } from 'knex';
+import { getHelpers } from '../helpers/index.js';
+import { createInspector } from '@directus/schema';
+
+export async function up(knex: Knex) {
+	const inspector = createInspector(knex);
+	const helper = getHelpers(knex).schema;
+	const csvFields = await knex.select('collection', 'field').from('directus_fields').where('special', '=', 'cast-csv');
+
+	const updates: Promise<void>[] = [];
+
+	for (const { collection, field } of csvFields) {
+		updates.push(
+			inspector.columnInfo(collection, field).then((column) => {
+				if (column.data_type === 'text') return;
```

Comment:
Would it be possible to check for `!== 'string'` instead here? 🤔

Reply:
Data type for "string" differs depending on the DB (`string`, `varchar`, ...) so `=== 'text'` is the easier way

Reply:
Yeah thats what i was afraid of